src/views/More/ShippingReturnPolicy/mock.ts

```diff
@@ -0,0 +1,20 @@
+export const list = [
+  {
+    title: "Warranty",
+    content:
+      "We offer replacement or repair for parts that are defected in materials and workmanship from the date of purchase for a period of 1 year. We will repair any broken parts of a product using new or replacement parts, or issue you store credit as a refund (except for a fee for shipping, handling, packing, return postage, and insurance which will be incurred by the customer). Such repair or replacement is subject to verification of the defect or malfunction and proof of purchase as confirmed by showing the order number on original dated sales receipt. In order to get the service, please refer to returning terms and contact customer service department first. This limited warranty does not cover normal wear and tear, damage or defects caused by accidents, misuse, abuse, or lack of necessary maintenance; nor does it cover alteration, modification, improper installation, improper handling/storage, environmental conditions, nor any labor expenses. Due to climate changes, heat or humidity can cause wood to expand/contract, swell/warp or even dry out. Since these are all natural causes and not manufacture defects, we are not responsible for any replacement or repair. This warranty does not cover the natural aging or darkening/mellowing of wood color, the inherent growth characteristics of, or variation in wood. Not two trees are alike, same for wood. The difference in color and pattern in wood are not considered as imperfection. This limited warranty is not transferable.To obtain warranty service, you must first contact us to determine the problem and the appropriate solution for you. Please refer to Returning Policies and feel free to contact customer service for supports.",
+    expanded: false,
```

Comment:
I don't think we need the `expanded` flag, since it's closed by default.

Reply:
Done